lib/rules/no-unsafe-innerhtml.js

```diff
@@ -70,7 +70,7 @@ module.exports = function (context) {
             } else {
                 allowed = false;
             }
-        } else if (expression.type === "CallExpression") {
+        } else if ((expression.type === "CallExpression") && (expression.callee.property)) {
             var funcName = expression.callee.name || expression.callee.property.name;
```

Comment:
If expression.callee.property doesn't exist, we can use expression.callee.name (see line 74).